e2e/scripts/common.func

```diff
@@ -0,0 +1,178 @@
+# generic functions
+
+fatal() {
+    test-error "$1"
+    exit 1
+}
+
+# test if singularity binary is present in $PATH
```

Comment:
Could you use your `check-command` built-in here?

Reply:
no because if singularity is not found, it's considered as fatal, `check-command` just skip the current test script